openat.cc

```diff
@@ -176,10 +176,14 @@ class OpenatTest : public ::testing::Test {
     // Create a couple of nested directories
     int rc = mkdir(TmpFile(TOPDIR), 0755);
     EXPECT_OK(rc);
-    if (rc < 0) EXPECT_EQ(EEXIST, errno);
-    rc = mkdir(TmpFile(SUBDIR_ABS), 0755);
+    if (rc < 0) {
+      EXPECT_EQ(EEXIST, errno);
+    }
+    rc = mkdir(TmpFile(SUBDIR), 0755);
```

Comment:
Looks like this PR depends on #35 ?

Reply:
Yeah, but I could shuffle stuff around so this one goes first.

Reply:
Oh, yeah.. let me fix that.

Reply:
Fixed by removing the accidental change from #35.